index.html

```diff
@@ -10,8 +10,8 @@
 humantime: "9:00 am - 4:00 pm"    # human-readable time (e.g., "9:00 am - 4:30 pm")
 startdate: 2016-04-25      # use YYYY-MM-DD format like 2015-01-01
 enddate: 2016-04-26      # use YYYY-MM-DD format like 2015-01-02
-instructor: ["Greg Wilson", "Madeleine Bonsma", "Elliott Sales de Andrade", "Erin Dickie"] # list of names like ["Kay McNulty", "Betty Jennings", "Betty Snyder"]
-helper: ["Sarah Meister", "Dawn Walker", "Heather Gibling", "Yohan Yee", "Jeff Wintersinger"]     # list of names like ["Marlyn Wescoff", "Fran Bilas", "Ruth Lichterman"]
+instructor: ["Greg Wilson", "Heather Gibling", "Elliott Sales de Andrade", "Erin Dickie"] # list of names like ["Kay McNulty", "Betty Jennings", "Betty Snyder"]
+helper: ["Sarah Meister", "Dawn Walker", "Yohan Yee", "Jeff Wintersinger", "Madeleine Bonsma", "Christian Farfan Centeno", "Zoe Francis", "John Ladan", "Natalia Mykhaylova", "Lindsay Coome", "Luke Johnston"]     # list of names like ["Marlyn Wescoff", "Fran Bilas", "Ruth Lichterman"]
```

Comment:
Woohoo, extra helpers!

Before you merge, could I get Christian's, John's, and Natalia's emails? For updating the workshop database.